testsuite/src/main/java/io/netty/testsuite/transport/socket/DatagramUnicastTest.java

```diff
@@ -267,20 +248,21 @@ public void channelRead0(ChannelHandlerContext ctx, Object msgs) throws Exceptio
         }
     }
 
-    private static ChannelFuture write(Channel cc, ByteBuf buf, InetSocketAddress remote, WrapType wrapType) {
+    protected ChannelFuture write(Channel cc, ByteBuf buf, SocketAddress remote, WrapType wrapType) {
```

Comment:
Looks like this can be abstract here, and the implementation moved to `DatagramUnicastInetTest`

Reply:
done